Doc/library/optparse.rst

```diff
@@ -11,8 +11,9 @@
 **Source code:** :source:`Lib/optparse.py`
 
 .. deprecated:: 3.2
-   The :mod:`optparse` module is deprecated and will not be developed further;
-   development will continue with the :mod:`argparse` module.
+   The :mod:`optparse` module is :term:`soft deprecated` and will not be
+   developed further; development will continue with the :mod:`argparse`
+   module.
```

Comment:
This claim remains in blatant contradiction with the conclusion of #53580 that “argparse is very difficult to change without risk of breaking many existing tools that depend on argparse working exactly the way it does now”.

If argparse has become too difficult to change, even to fix known parsing bugs, then either argparse should be soft-deprecated for the same reason as optparse, or optparse should be un-soft-deprecated.

Reply:
I think the optparse soft deprecation should be reversed because of how argparse is.